client/signup/steps/user/index.jsx

```diff
@@ -149,6 +150,7 @@ export class UserStep extends Component {
 				submitForm={ this.submitForm }
 				submitButtonText={ this.submitButtonText() }
 				suggestedUsername={ this.props.suggestedUsername }
+				isSocialSignupEnabled={ isEnabled( 'signup/social' ) }
```

Comment:
Instead of adding this by default to every user step in development, it would probably be a little bit less disruptive to create a separate, development-only flow for testing social signup with a version of this step with `isSocialSignupEnabled` set to true. This way, there isn't any added confusion about broken signup buttons in `/start` in development. What do you think?